develop/debugging/pdb.rst

```diff
@@ -229,3 +238,93 @@ debug prompt.
 For more information see
 
 * http://docs.python.org/library/pdb.html
+
+
+Interactive debugging in your ``bin/{client1|instance}`` (command line)
+=======================================================================
+
+You can use interactive debugging via ``bin/{client1|instance} debug`` (use the name of the instance script you're using in your buildout). It gives you an interactive Python interpreter with access to Zope's root object (bound to "app"). In the interpreter, you can do "normal" Python debugging.
+
+
+Alternatives to ``pdb``
+=======================
+
+Some of these options (like ``q``) are complementary to ``pdb`` itself. We suggest you to try the alternatives here, some features (like tab completion and syntax highlighting) are really hard to live without after getting used to them.
+
+ipdb
```

Comment:
Can you tell me exactly what problems we have with `ipdb`? I just tested in a Plone 5 instance and it looks fine...

Reply:
Some month ago I run into some dependecy/version hell. Then I switched to pdbpp and I am happy with it. So, if it now works its fine and lets keep it.

Reply:
I much prefer `pdbpp` over `ipdb`. At least I would switch the order to mention `pdbpp` first. Also I would mention that `iw.debug` is obsolete when you use Products.PDBDebugMode because is has a view `pdb`.

Reply:
The reST syntax here needs lots of improvements and besides that is is not following our style-guide at all ! :)

Reply:
Can you help where is not following the reST syntax? When I go to https://docs.plone.org/develop/styleguide/documentation.html there's no specific details about how to use the reST markup...

Reply:
Here: https://docs.plone.org/about/contributing/documentation_styleguide.html